de.json

```diff
@@ -1,132 +1,132 @@
 {
-    "no-admin": "❌Sie sind kein Admin",
```

Comment:
"Sie" (you) is formal and "Du" (you) is informal
https://en.wiktionary.org/wiki/Sie